packages/server/lib/util/env-editors.ts

```diff
@@ -129,7 +129,7 @@ const windowsEditors = [
     name: 'Brackets',
   }, {
     id: 'code',
-    binary: 'Code.exe',
+    binary: 'code',
```

Comment:
weird that there's no `.exe`, but was there one before? i.e. is this going to break code detection for older vscode users?

Reply:
Pft, I dunno, was going to ask @chrisbreiding how he determined the list to begin with.

Reply:
I don't remember exactly where this list came from. I thought I had left comments in the code pointing the source, but I guess not. I think I compiled it from a few different sources, like [env-editor](https://github.com/sindresorhus/env-editor), but I'm not sure where precisely I got the VSCode binary value for Windows. I didn't manually test out each editor on each OS, so it's possible it has always been wrong, or it's equally possible it has changed at some point from `Code.exe` to `Code` since this was implemented.